spec/bot_spec.rb

```diff
@@ -6,7 +6,8 @@
 
 describe Bot do
   let(:action)                { "created" }
-  let(:title)                 { "[#LIEF-123] Cure World Hunger!" }
+  let(:title)                 { "[#LIEF-1234] Cure World Hunger!" }
+  let(:branch_name)           { "feature/lief 1234 cure world hunger!" }
```

Comment:
Are you sure that's the data we become from github? Our branchnames are generally something like "feature/lief-1234-cure-world-hunger!"